pkgs/development/python-modules/pytest-order/default.nix

```diff
@@ -0,0 +1,37 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytest
+, pytest-xdist
+, pytest-dependency
+, pytest-mock
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-order";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "mrbean-bremen";
+    repo = "pytest-order";
+    rev = "486e0966c651f3c57bbc2172e9914e478de55b6b";
+    sha256 = "1mfw2x8r5hn4g5cd6jca92lvqrd9297gsc5bvz18dzjlrm8zlycc";
+  };
+
+  propagatedBuildInputs = [
+    pytest
+    pytest-xdist
+    pytest-dependency
+    pytest-mock
```

Comment:
Only `pytest` should be in `propagatedBuildInputs`: https://github.com/mrbean-bremen/pytest-order/blob/v0.9.3/setup.py#L30
The rest should be put into `checkInputs`.